x-pack/plugins/lists/public/exceptions/hooks/use_api.ts

```diff
@@ -105,6 +110,62 @@ export const useApi = (http: HttpStart): ExceptionsApi => {
           onError(error);
         }
       },
+      async getExceptionListsItems({
+        lists,
+        filterOptions,
+        pagination,
+        showDetectionsListsOnly,
+        showEndpointListsOnly,
+        onSuccess,
+        onError,
+      }: ApiCallFindListsItemsMemoProps): Promise<void> {
+        const abortCtrl = new AbortController();
+        const { ids, namespaces } = lists
+          .filter((list) => {
+            if (showDetectionsListsOnly) {
+              return list.type === 'detection';
+            } else if (showEndpointListsOnly) {
+              return list.type === 'endpoint';
+            } else {
+              return true;
+            }
+          })
+          .reduce<{ ids: string[]; namespaces: NamespaceType[] }>(
+            (acc, { listId, namespaceType }) => ({
+              ids: [...acc.ids, listId],
+              namespaces: [...acc.namespaces, namespaceType],
+            }),
+            { ids: [], namespaces: [] }
+          );
+
+        try {
+          if (ids.length > 0 && namespaces.length > 0) {
```

Comment:
What happens if they are not > 0?

Reply:
Just doesn't make the call here since there would be no lists to search.

Reply:
Updated so if ids are not > 0 then it still calls onSuccess but with defaults.